### PR TITLE
fix: use correct type for non-toJSON js/css on HttpIncoming

### PR DESCRIPTION
**Pull request by wkillerud**
Relates to https://github.com/podium-lib/podlet/pull/410

### CODE REVIEW

File: tests/asset-css.test.js [hunk 1]
```diff
@@ -477,6 +477,7 @@ tap.test('Css() - set "href" - should throw', (t) => {
 
 tap.test('Css() - validate object against schema - should validate', (t) => {
     const obj = new AssetCss({ value: '/foo' });
+    // @ts-expect-error TODO: should the schema accept boolean for crossorigin?
```
Comment (wkillerud) [inline tests/asset-css.test.js:480]: Not sure if we should update the schema here or not

Comment (digitalsadhu) [reply tests/asset-css.test.js:480]: hmm good question, its boolean at the object level true, but this is the same as crossorigin="". Feels like a the schema level, it should follow the spec https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/crossorigin
 "", "use-credentials" or "anonymous"

Comment (wkillerud) [inline tests/asset-css.test.js:480]:
```suggestion
    // @ts-expect-error Object model w/ `boolean` doesn't match the spec (#255)
```

File: tests/asset-js.test.js [hunk 2]
```diff
@@ -455,6 +455,7 @@ tap.test('Js() - set "src"', (t) => {
 
 tap.test('Js() - validate object against schema - should validate', (t) => {
     const obj = new AssetJs({ value: '/foo' });
+    // @ts-expect-error TODO: should the schema accept boolean for crossorigin?
```
Comment (wkillerud) [inline tests/asset-js.test.js:458]:
```suggestion
    // @ts-expect-error Object model w/ `boolean` doesn't match the spec (#255)
```

**Pull request closed without merging**